### PR TITLE
update graphic files that show Insolar Wallet interface, update text …

### PR DESCRIPTION
**Pull request by raproid**
…to comply with Insolar documentation style guide

### CODE REVIEW

**Comment by raproid** [comment]
conflicts resolved

File: docs/source/ledger-nano.rst [hunk 1]
```diff
@@ -77,19 +77,23 @@ To install the Insolar application on Ledger Nano:
    .. image:: imgs/nano/ledger-live-connect.png
       :width: 600px
 
+   .. caution::
+
+      If you're using Ledger Nano S, make sure your Micro-USB cable supports both charging and data transfer. If it's a charge-only cable, you won't be able to complete this guide.
```
Comment (sofiayem) [inline docs/source/ledger-nano.rst]: The user does not care about guide completion. He/she wants to fix a problem or perform an action and, if he/she can't, wants to know why. Suggestion: If it's a charge-only cable, your computer won't recognize the device and you won't be able to perform any action.

File: docs/source/ledger-nano.rst [hunk 2]
```diff
@@ -193,18 +197,13 @@ Receive XNS
...
-#. Open the dashboard of the Insolar Wallet and click the avatar icon the in upper-right corner.
-
-   .. image:: imgs/nano/click-avatar.png
-      :width: 250px
+#. On the dashboard of your wallet, click the copy button below :guilabel:`XNS account`. This copies your XNS account address to the clipboard.
 
-#. In the **Your Wallet** panel, click :guilabel:`Copy XNS address`. This copies the address to the clipboard.
-
-   .. image:: imgs/nano/copy-xns-address.png
+   .. image:: imgs/nano/copy-xns-account-address.png
```
Comment (sofiayem) [inline docs/source/ledger-nano.rst:202]: I'd rather blur all addresses. Yeah, I know that that's on TestNet but still, for the sake of consistency.

File: docs/source/swap.rst [hunk 3]
```diff
@@ -66,31 +66,33 @@ To swap your INS tokens to XNS coins and simultaneously migrate them from the Et
...
+   * Click :guilabel:`SAVE TO BROWSER` to save it to your browser local storage. Keeping the file locally allows easier access from the browser on the device you are using.
+   * Click :guilabel:`DOWNLOAD` to save it to your device. In this case, you can later move it to another device via, for example, a USB drive.
 
    Later, you can log in using one of the following:
 
-   * (Recommended) Your password and the keystore file.
-   * Unencrypted private key.
+   * (Recommended) Your password and the keystore file saved to your browser.
+   * Your password and the keystore file saved to your device.
+   * Hardware wallet (such as Ledger Nano X or S).
```
Comment (sofiayem) [inline docs/source/swap.rst]: Now, I'd rather recommend the hardware option. Second is pass and keystore saved to device. Third—to browser.

File: docs/source/test-swap.rst [hunk 4]
```diff
@@ -30,11 +30,11 @@ Test scenario overview
 
 To grasp what you will be doing, take a look at the steps of the test scenario:
 
-#. Create a MetaMask wallet and, inside, switch to Ropsten test network and add the INS custom token.
+#. Create a MetaMask wallet and, inside of it, switch to Ropsten test network and add the INS custom token.
```
Comment (sofiayem) [inline docs/source/test-swap.rst]: "... of it" feels kinda off to me.

File: docs/source/test-swap.rst [hunk 5]
```diff
@@ -244,35 +244,34 @@ To migrate the test INS tokens and swap them to XNS coins:
...
 
-   .. |avatar-icon| image:: imgs/mig-test/avatar-icon.png
-      :width: 30px
-
-   .. image:: imgs/mig-test/right-menu.png
-      :width: 200px
+   .. image:: imgs/mig-test/wlt-test-open-swap-tab.png
+      :width: 600px
 
    In the menu, click :guilabel:`Copy migration address`.
```
Comment (sofiayem) [inline docs/source/test-swap.rst]: It's not the menu now, is it? It's the screen of the SWAP tab.

File: docs/source/use-cases.rst [hunk 6]
```diff
@@ -189,13 +189,13 @@ Use case: Transfer coins
 
 To transfer XNS coins to another user, a user (exchange or any other) can either:
 
-* Use the Insolar's Web Wallet.
+* Use their Insolar Wallet.
```
Comment (sofiayem) [inline docs/source/use-cases.rst:192]: There's an emphasis on Web here. I mean the UI as opposed to API. Suggestion: Use the web UI of their wallet provided by Insolar.

File: docs/source/use-cases.rst [hunk 7]
```diff
@@ -248,38 +248,38 @@ API requests used:
 
 .. _deposit_funds_to_ex:
 
-Use case: Deposit funds to exchange
+Use case: Deposit funds to an exchange
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (sofiayem) [inline docs/source/use-cases.rst]: Let's not make sphinx too nervous.

/Users/syem/ins-docs/docs/source/use-cases.rst:252: WARNING: Title underline too short.

Use case: Deposit funds to an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:252: WARNING: Title underline too short.

Use case: Deposit funds to an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:264: WARNING: Title underline too short.

Use case: Deposit coins to an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:264: WARNING: Title underline too short.

Use case: Deposit coins to an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:276: WARNING: Title underline too short.

Use case: Withdraw coins from an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/syem/ins-docs/docs/source/use-cases.rst:276: WARNING: Title underline too short.

Use case: Withdraw coins from an exchange
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Review by sofiayem — CHANGES_REQUESTED** [review]

File: docs/source/ledger-nano.rst (hunk 1 above)
Comment (raproid) [reply docs/source/ledger-nano.rst]: done

File: docs/source/ledger-nano.rst (hunk 2 above)
Comment (raproid) [reply docs/source/ledger-nano.rst:202]: I didn't? F-fffuuu...

File: docs/source/swap.rst (hunk 3 above)
Comment (raproid) [reply docs/source/swap.rst]: done

File: docs/source/test-swap.rst (hunk 5 above)
Comment (raproid) [reply docs/source/test-swap.rst]: done

Comment (raproid) [reply docs/source/test-swap.rst]: On the `SWAP`tab, click :guilabel:`Copy migration address`.

Comment (raproid) [reply docs/source/test-swap.rst]: On the :guilabel:`SWAP` tab, click :guilabel:`Copy migration address`.

File: docs/source/test-swap.rst (hunk 4 above)
Comment (raproid) [reply docs/source/test-swap.rst]: , inside, doesn't look very intuitive either.

Comment (raproid) [reply docs/source/test-swap.rst]: Create a MetaMask wallet, switch to Ropsten test network inside it, and add the INS custom token. - how about this?

File: docs/source/use-cases.rst (hunk 6 above)
Comment (raproid) [reply docs/source/use-cases.rst:192]: Use the web UI of their wallet provided by Insolar. — way too long

Comment (raproid) [reply docs/source/use-cases.rst:192]: Isn't "Insolar Wallet" associated with the web version of the product? We have it on so many screenshots and in docs. Shall we, maybe, explicitly call the API "Insolar Wallet API" and leave "Insolar Wallet" attached to the web version?

File: docs/source/use-cases.rst (hunk 7 above)
Comment (raproid) [reply docs/source/use-cases.rst]: fixed all the cases in use-cases.rst

File: docs/source/ledger-nano.rst (hunk 2 above)
Comment (raproid) [reply docs/source/ledger-nano.rst:202]: fixed

File: docs/source/test-swap.rst (hunk 4 above)
Comment (raproid) [reply docs/source/test-swap.rst]: fixed according to our Slack discussion

File: docs/source/use-cases.rst (hunk 6 above)
Comment (raproid) [reply docs/source/use-cases.rst:192]: Use their Insolar Wallet web interface

**Review by sofiayem — APPROVED** [review]

**Pull request merged**